lib/modules/manager/cpanfile/common.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { extractPerlVersion } from './common';
+
+describe('modules/manager/cpanfile/common', () => {
+  describe('extractPerlVersion', () => {
+    test.each`
+      version         | expected
+      ${5.012005}     | ${'5.012005'}
```

Comment:
```suggestion
      ${'5.012005'}     | ${'5.012005'}
```

Reply:
I understand, I just want it to be inserted into the source string as the string value, not JavaScript float